common/src/main/java/com/wynntils/features/chat/ChatItemFeature.java

```diff
@@ -249,26 +273,14 @@ private List<StyledTextPart> createItemPart(WynnItem wynnItem) {
         return parts;
     }
 
-    private static void makeChatPrompt(Slot hoveredSlot) {
-        // Special case for unidentified gear
-        Optional<GearItem> gearItemOpt = Models.Item.asWynnItem(hoveredSlot.getItem(), GearItem.class);
-        if (gearItemOpt.isPresent() && gearItemOpt.get().isUnidentified()) {
-            // We can only send chat encoded gear of identified gear
-            WynntilsMod.warn("Cannot make chat link of unidentified gear");
-            McUtils.sendErrorToClient(I18n.get("feature.wynntils.chatItem.chatItemUnidentifiedError"));
-            return;
-        }
-
-        Optional<WynnItem> wynnItemOpt = Models.Item.getWynnItem(hoveredSlot.getItem());
-        if (wynnItemOpt.isEmpty()) return;
-
-        // TODO: These settings should be persisted, and changeable in the custom share GUI
-        EncodingSettings encodingSettings = new EncodingSettings(false, true);
+    private void makeChatPrompt(WynnItem wynnItem, Optional<GearItem> gearItemOpt) {
```

Comment:
I think it would be cleaner if you just got thte gearItemOpt again before it's used, instead of passing it down